pkg/reconciler/knativeserving/knativeserving_controller.go

```diff
@@ -233,6 +225,63 @@ func (r *ReconcileKnativeServing) install(instance *servingv1alpha1.KnativeServi
 	return nil
 }
 
+func (r *ReconcileKnativeServing) executeInstall(extensions common.Extensions, instance *servingv1alpha1.KnativeServing) error {
+	err := r.config.Transform(extensions.Transform(r.scheme, instance)...)
+	if err != nil {
+		return err
+	}
+	err = extensions.PreInstall(instance)
+	if err != nil {
+		return err
+	}
+	err = r.applyAllResources(instance)
+	if err != nil {
+		return err
+	}
+	return extensions.PostInstall(instance)
+}
+
+func (r *ReconcileKnativeServing) applyAllResources(instance *servingv1alpha1.KnativeServing) error {
```

Comment:
[nit] applyEnabledResources or applyIncludedResources?

Otherwise r.applyAllResources and  config.ApplyAll is confusing to differentiate.

Reply:
Done.